dev-python/panel/panel-0.11.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+# Panel imports from "distutils" at runtime.
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="High-level app and dashboarding solution for Python"
+HOMEPAGE="
+	https://panel.holoviz.org
+	https://pypi.org/project/panel
+	https://github.com/holoviz/panel"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+BDEPEND="
+	>=dev-python/setuptools-30.3.0[${PYTHON_USEDEP}]
+	>=net-libs/nodejs-15.11.0
+"
+DEPEND="
+	>=dev-python/bokeh-2.3.0[${PYTHON_USEDEP}]
+	>=dev-python/param-1.10.0[${PYTHON_USEDEP}]
+	>=dev-python/pyct-0.4.4[${PYTHON_USEDEP}]
+"
+RDEPEND="${DEPEND}
+	dev-python/markdown[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/tqdm[${PYTHON_USEDEP}]
+	>=dev-python/pyviz_comms-0.7.4[${PYTHON_USEDEP}]
+"
+
+# Permit Panel to violate network sandboxing, as "setup.py" invokes "npm" to
+# fetch and bundle Node.js assets with Panel.
+RESTRICT=network-sandbox
```

Comment:
We're going to add this to [EAPI 8 ](https://wiki.gentoo.org/wiki/Future_EAPI/EAPI_8_tentative_features)but for now, we need just RESTRICT="test" to be [PMS-compliant](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-680007.3.6).

Reply:
I'll patch that right up. Thanks for those topical links, too. [I'm with Górny](https://bugs.gentoo.org/553696#c5):

> Let's extend `RESTRICT` syntax to make it `token[:reason]`, e.g. `test:network`, `test:docker`. The idea behind that is that the first token is well-defined and always work, while the second one can have implementation-defined meaning. We can then add new reasons without any changes to the PMS, and old PMs will work just fine (assuming `RESTRICT=test` is effective).